app/movie/page.tsx

```diff
@@ -1,5 +1,5 @@
 import { redirect } from "next/navigation";
```

Comment:
question: What is this file for?

Reply:
if users type like 'http://localhost:3002/movie' it will redirect to http://localhost:3002/